data/anaconda.conf

```diff
@@ -76,6 +76,14 @@ enabled_repositories_from_treeinfo = addon optional variant
 # Enable installation from the closest mirror.
 enable_closest_mirror = True
 
+# Enable %packages --ignorebroken feature.
+# This will automatically skip conflicting and broken packages. This may result
+# in getting installed system which won't have everything installed as expected.
+# Because of this problem this feature will be disabled for some platforms. If disabled
+# the installation won't be started, instead it will inform user that the feature is not
+# supported on their platform.
```

Comment:
I don't think that this super long explanation needs to be here. This file should be easy to read. Also, it is not clear whether you are talking about the configuration option or the kickstart option. I would move the explanation to `rhel.conf` to document why we have decided to disable it in RHEL.

Reply:
Reworded.